cabal.project

```diff
@@ -66,13 +66,20 @@ optional-packages:
 
 allow-newer:
   brick:base,
+  brick:deepseq,
```

Comment:
I think we can drop the following ones:
```
vector-binary-instances:base,
cryptohash-sha256:base,
haskell-src-meta:template-haskell,
string-interpolate:template-haskell,
string-interpolate:text,
tasty-hedgehog:tasty,
```

Reply:
Perhaps that also goes for `clash-ffi/example/cabal.project`? I did not test it.

Reply:
Thanks, I've made the allow-newer lists smaller.